test/server/config_validation/BUILD

```diff
@@ -89,13 +78,12 @@ envoy_cc_fuzz_test(
     corpus = "config_corpus",
     deps = [
         "//source/common/thread_local:thread_local_lib",
+        "//source/server:configuration_lib",
+        "//source/server:options_lib",
         "//source/server:server_lib",
+        "//source/server/config_validation:server_lib",
         "//test/integration:integration_lib",
         "//test/mocks/server:server_mocks",
-        "//test/mocks/stats:stats_mocks",
-        "//source/server:configuration_lib",
-        "//source/server/config_validation:server_lib",
-        "//source/server:options_lib",
         "//test/test_common:environment_lib",
```

Comment:
In general, these deps should reflect the headers in `config_fuzz_test.cc`; can you clean this up?